lib/toxiproxy.rb

```diff
@@ -54,6 +53,11 @@ def self.all
     Collection.new(proxies)
   end
 
+  # Sets the toxiproxy host to use.
+  def self.host=(host)
+    @uri = host && ::URI.parse(host)
```

Comment:
If the host is a `URI`, we shouldn't parse it. There's no need to check if it's set, it'll always be set since it's a mandatory argument.